docs/api-reference/endpoints/status/index.md

```diff
@@ -7,8 +7,13 @@ This endpoint is not available in the public API, but you can use it when runnin
 The GET request http://localhost:8082/ors/v2/status (host and port are dependent on the setup) returns basic information about the running instance:
 
 * `languages`: available languages
-* `engine`: the build date and version of the openrouteservice
-* `profiles`: available routing profiles, info about storages and configured limits
+* `engine`: the build date and version of the openrouteservice.
+  `graph_version` is the version of the graph structure and build logic determining the graph repository address
+  if [graph management](/run-instance/configuration/engine/graph-management.md) is enabled
+* `profiles`: available routing profiles,
+  info about the graph's build (start) date (`graph_build_date`),
+  timestamp of the osm pbf file that was used for building the graph (`osm_date`),
```

Comment:
I feel like we should explain these two in more detail, as to not get people confused.

Reply:
OK, I tried to explain a little bit more... is it understandable?